scripts/tools/zap_regen_all.py

```diff
@@ -292,8 +292,10 @@ def formatKotlinFiles(self, paths):
             jar_url = f"https://repo1.maven.org/maven2/com/facebook/ktfmt/{VERSION}/{JAR_NAME}"
 
             with tempfile.TemporaryDirectory(prefix='ktfmt') as tmpdir:
-                path, http_message = urllib.request.urlretrieve(jar_url, Path(tmpdir).joinpath(JAR_NAME).as_posix())
-                subprocess.check_call(['java', '-jar', path, '--google-style'] + paths)
+                path, http_message = urllib.request.urlretrieve(
```

Comment:
Can we minimize the diff due to formatting? Is our restyler different than on CSA ?

Reply:
will do, I assumed this hadn't been restyled in a while but it is possible our python formater is different...